### DISCUSSION TITLE
Difference between complete, voxelwise, and depthBased heterogeneity correction

### DISCUSSION THREAD

**Discussion by saxu3829** [e0404/matRad#708] [Q&A] [answered]
Hey, 
I am currently working on understanding the heterogeneity correction in the research branch. If I understood correctly complete performs a correction on every point in the phantom, whereas voxelwise discerns the phantom in voxels to perform the same correction on this group of points. Lastly, depthBased then only looks at the depth. The convolution should be most accurate in case of the complete type, am I right? When I use a single pencil beam to look at the differences between the different types of correction, patterns can be observed that seem to occur from this discretization. Is there any way to know when to use which type of correction or is it only dependend on the desired computation time and modulation/dose accuracy of the investigated cases?

**Comment by wahln** [comment]
Sorry for not getting back to you here for a long time. @HomolkaN should now about this in most detail.

**Comment by HomolkaN** [comment, marked as answer]
yes I do. Complete correction is using a single Gaussian to convolve the anyltical depth dose, voxel-wise does the same but with individual Gaussians for each voxel where there's a dose distribution affected by lung tissue and depth-dependent uses different Gaussians for each different sub-component of the depth dose.
This means that for a phantom, where the target is some distance away from the lung tissue, voxel-wise and complete should be identical. 

That said, voxel-wise is the most precise method of the 3 implemented ones, especially for targets in or close to lung tissue. It is therefore used as the default method. Hope I could help.
